packages/core-data/src/queried-data/reducer.js

```diff
@@ -138,6 +140,35 @@ const queries = flowRight( [
 	);
 } );
 
+/**
+ * Reducer tracking queries state.
+ *
+ * @param {Object} state  Current state.
+ * @param {Object} action Dispatched action.
+ *
+ * @return {Object} Next state.
+ */
+const queries = ( state = {}, action ) => {
+	switch ( action.type ) {
+		case 'RECEIVE_ITEMS':
+			return receiveQueries( state, action );
+		case 'REMOVE_ITEMS':
+			const newState = { ...state };
+			const removedItems = action.itemIds.reduce( ( result, itemId ) => {
+				result[ itemId ] = true;
+				return result;
+			}, {} );
+			forEach( newState, ( queryItems, key ) => {
+				newState[ key ] = filter( queryItems, ( queryId ) => {
+					return !! removedItems[ queryId ] === false;
```

Comment:
This is coercing a boolean to a boolean.

Reply:
No, we're making undefined false. I am keeping (filter) everything that is not in `removedItems` and hence check for `!! removedItems[ queryId ] === false` and I am interested in when `removedItems[ queryId ]` is undefined.

Reply:
Then you could just return `removedItems[ queryId ]`.

Reply:
```
const removedItems = { a: true, c: true };
const newState = [ "a", "b" ];
console.log( _.filter( newState, ( queryId ) => {
  return removedItems[ queryId ];
} ) );
=> 
["a"]
```
I want `["b"]` of that, so `! removedItems[ queryId ]`, still simpler then my expression, Thanks!